src/pages/docs/liveobjects/storage.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: Object storage
+meta_description: "Learn about LiveObjects object storage."
+---
+
+<Aside data-type='experimental'>
+LiveObjects is currently Experimental. Its features are still in development and subject to rapid change.
+
+**Building with LiveObjects?** Help shape its future by [sharing your use case](https://44qpp.share.hsforms.com/2fZobHQA1ToyRfB9xqZYQmQ).
+</Aside>
+
+## Default object storage - 90 days <a id="default-persistence"/>
+
+Ably durably stores all objects on a channel for a retention period that is configured to 90 days by default. If the data is not updated within the retention period, it will automatically expire. After expiry, the channel is reset to its initial state and only includes an empty [root object](/docs/liveobjects/concepts/objects#root-object).
+
+<Aside data-type='note'>
+If you're interested in a configurable default object storage duration, please [get in touch](/contact) with your requirements.
+</Aside>
+
+## Store objects outside of Ably <a id="outside-persistence"/>
+
+You can store your objects outside of Ably by obtaining the channel objects via the [REST API](/docs/liveobjects/rest-api-usage#fetching-objects) and storing them in your own database. This is useful if you want to keep a permanent record of the objects on a channel or if you want to perform analytics on the data.
+
+In order to receive notifications when the objects on a channel are updated, use [inband objects](/docs/liveobjects/inband-objects) to receive updates as regular channel messages.
+
+<Aside data-type='note'>
+If you're interested in exporting objects to your own systems using [integration rules](/docs/integrations/webhooks), please [get in touch](/contact) with your requirements.
```

Comment:
Broken link. Do we want to link to a [form](https://44qpp.share.hsforms.com/2fZobHQA1ToyRfB9xqZYQmQ) or [contact](https://ably.com/contact)??